### DISCUSSION TITLE
Blog grid layout?

### DISCUSSION THREAD

**Discussion by reeceatkinson** [facebook/docusaurus#7245] [Q&A] [answered]
Is it possible to show blog posts in a grid? If so how would I go about it?

**Comment by slorber** [comment, marked as answer]
@Josh-Cena display blog posts as grid: https://joshcena.com/blog/

<img width="1885" alt="CleanShot 2022-04-27 at 16 20 50@2x" src="https://user-images.githubusercontent.com/749374/165540338-fbf65d5f-edbb-48a7-820d-acece27d3175.png">


You can swizzle the blog list page and customize it the way you want it to be. 

https://github.com/Josh-Cena/Josh-Cena.github.io/blob/master/src/theme/BlogListPage/index.tsx

It can be configured so that there's no pagination and the list component receives all the blog posts as props.

**Reply by slorber** [reply]
@reeceatkinson that's the reality: if you swizzle an unsafe component, you risk breaking changes on upgrades

Now it's up to you:
- decide to swizzle and customize
- stay safe and keep the default UI

How would you present this choice differently?
